app/frontend/components/domains/permit-application/revision-sidebar.tsx

```diff
@@ -126,17 +173,33 @@ export const RevisionSideBar = observer(
           />
           {onCancel && (
             <Button variant="secondary" onClick={onCancel}>
-              {t("ui.cancel")}
+              {t("ui.close")}
             </Button>
           )}
         </Flex>
       )
     }
 
+    const selectedTabStyles = {
+      borderLeft: "1px Solid",
+      borderRight: "1px Solid",
+      borderTop: "4px solid",
+      borderColor: "border.dark",
+      borderLeftColor: "border.dark",
+      borderTopColor: "theme.blueAlt",
+      borderBottomColor: "theme.yellowLight",
+      borderRadius: 0,
+    }
+    const sortedPastRevisionRequests = Array.from(
+      (selectedPastSubmissionVersion?.revisionRequests as IRevisionRequest[]) ?? []
+    ).sort((a, b) => {
+      return a.createdAt - b.createdAt
+    })
```

Comment:
maybe use a use memo, otherwise the sort operation will run in every render cycle which is unnecessary in this case